test/release/examples/primers/classes.chpl

```diff
@@ -63,14 +80,54 @@ class D: C {
 // Because the class ``D`` is derived from ``C``, the variable ``foo`` can
 // reference an object of type ``D``.  If an overridden method such as
 // ``printFields`` is called, it is dynamically dispatched to the method
-// with the most specific dynamic type.  The ``delete`` keyword can be
-// used to free memory associated with ``foo`` before pointing it an instance
-// of ``D``.
+// with the most specific dynamic type.
 //
-delete foo;
-foo = new unmanaged D(3, 4);
+// Note that since ``foo`` is an ``owned C``, assigning to it
+// will delete the previous instance "owned" by that variable.
+foo = new owned D(3, 4);
 foo.printFields();
-delete foo;
+
+
+// A class type includes a memory management strategy. The currently supported
+// strategies are ``owned``, ``shared``, ``unmanaged``, and ``borrowed``.
+var unm: unmanaged C = new unmanaged C();
+// 'unm' refers to a manually managed instance. It needs to have ``delete``
+// called on it to free the memory.
+delete unm;
+
+var own: owned C = new owned C(1, 10);
+// the instance referred to by 'own' is deleted at end of scope
```

Comment:
Maybe "at the end of *this* scope" (emphasis to indicate what was changed)

Reply:
Or "when it is no longer in scope".  "Of scope" just sounds a little weird to me, idk